q2_types/_feature_data.py

```diff
@@ -153,18 +161,20 @@ def tabular_msa_to_aligned_dna_sequences(view, data_dir):
 plugin.register_data_layout(taxonomy_data_layout)
 plugin.register_data_layout_reader('taxonomy', 1, pd.Series,
                                    taxonomy_to_pandas_series)
+plugin.register_data_layout_reader('taxonomy', 1, pd.DataFrame,
+                                   taxonomy_to_pandas_dataframe)
 plugin.register_data_layout_reader('taxonomy', 1, qiime.Metadata,
                                    taxonomy_to_qiime_metadata)
```

Comment:
Would adding a `.MetadataCategory` reader make sense, just to balance out the two choices for each `pd` layout? I can definitely see where `.Metadata` could be made to do some cool pre-made `df` recipe type things, and same goes for the `.MetadataCategory`.